tests/test_pyside_renderer.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+
+
+try:
+    from PySide6 import QtCore, QtWidgets
+except ImportError:
+    pytest.skip(
+        "skip test for PySide6 renderer when not available", allow_module_level=True
+    )
+
+
+from collagraph import Collagraph, create_element as h, EventLoopType
+from collagraph.renderers import PySideRenderer
+
+
+# Make sure a Qt application already exists
+QtCore.QCoreApplication.instance() or QtWidgets.QApplication()
```

Comment:
You may want to put this in a fixture so that it's executed as part of the test suite, not as part of importing and test collection:

```py
@pytest.fixture(autouse=True, session=True)
def ensure_qt_app():
    QtCore.QCoreApplication.instance() or QtWidgets.QApplication()
```

You could move it to `conftest.py` as well, if you like.

Reply:
I've got something in place now that will do some clean-up after each test and checks that there are no left-over widgets from previous tests.